docs/docs/opentelemetry-langfuse-exporter.md

```diff
@@ -59,6 +59,58 @@ fun main() = runBlocking {
 ```
 <!--- KNIT example-langfuse-exporter-01.kt -->
 
+## Trace attributes
+
+Langfuse uses trace-level attributes to enhance observability with features like sessions, environments, tags and other metadata.
+The `addLangfuseExporter` function supports a `traceAttributes` parameter that accepts a list of `CustomAttribute` objects.
+
+These attributes are added to the root span of each trace and enable Langfuse's advanced features. You can pass
```

Comment:
I would also consider specifying here that we add this specifically to the `InvokeAgentSpan`.